src/mlx_omni_server/rerank/jina_mlx_reranker.py

```
@@ -0,0 +1,249 @@
+def sanitize_input(text: str, special_tokens: Dict[str, str]) -> str:
+    """Remove special tokens from input text."""
+    for token in special_tokens.values():
+        text = text.replace(token, "")
+    return text
+
+
+def format_docs_prompts_func(
+    query: str,
+    docs: list[str],
+    instruction: Optional[str] = None,
+    special_tokens: Dict[str, str] = {},
```

Ruff (0.14.5)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)